naughty/rhel4edge/4829-podman-hang

```diff
@@ -0,0 +1,3 @@
+  File "*", line *, in testPruneUnusedContainersSystem
```

Comment:
You could have unified the two patterns with `testPruneUnusedContainers*`, but this is fine of course.

Reply:
Yup, I chose to make it explicit as that's a bit easier.